package.json

```diff
@@ -13,6 +13,8 @@
     "url": "https://github.com/taskcluster/taskcluster-queue.git"
   },
   "dependencies": {
+    "ajv": "^4.2.0",
+    "ajv-keywords": "^1.1.1",
```

Comment:
Why do we need them here?
Shouldn't they be dependencies of `taskcluster-lib-validate`

And wouldn't it be enough to list `ajv-keywords` in-order to satisfy the peerDependency?

Hmm... If this is a new change in ajv, perhaps we should file a bug with ajv, given that adding a peerDependency is a breaking change. So they should have upgraded to a new major version.
Before we file that we should make sure that ajv did in fact change from not having a peerDep to having a peerDep in the 4.x series.